ci/aws-analytical-env/jobs/mirror-git-repo-s3.yml

```diff
@@ -0,0 +1,19 @@
+jobs:
+  - name: mirror-git-repo-s3
```

Comment:
you might want the repo name in the job/task, especially if there is intention to mirror additional repos

Reply:
Amended